docs/sphinx/developers/java-library.txt

```diff
@@ -60,13 +60,48 @@ all be found in our :sourcedir:`Git repository <jar>`.
 Examples of usage
 -----------------
 
+**Reading and Performance:**
```

Comment:
These should really be formatted as subheadings (underline using ^^^^^) rather than written in bold